src/components/cylc/tree/nodes.js

```diff
@@ -343,24 +343,11 @@ function createJobNode (job) {
   }
 }
 
-/**
- * @param {?WorkflowGraphQLData} workflow
- * @returns {boolean}
- */
-function containsTreeData (workflow) {
-  return workflow !== undefined &&
-    workflow !== null &&
-    workflow.cyclePoints && Array.isArray(workflow.cyclePoints) &&
-    workflow.familyProxies && Array.isArray(workflow.familyProxies) &&
-    workflow.taskProxies && Array.isArray(workflow.taskProxies)
-}
-
 export {
   createWorkflowNode,
   createCyclePointNode,
   createFamilyProxyNode,
   createTaskProxyNode,
   createJobNode,
-  containsTreeData,
```

Comment:
Not used. Or, we could — in theory — use it to filter out empty deltas, see #689 But then TUI would have to do something similar. I think it would be best if we didn't receive any deltas after `shutdown: true`, unless it's a new initial burst.